source/common/modules/markdown-editor/context-menu/default-menu.ts

```diff
@@ -21,6 +21,10 @@ import { type SyntaxNode } from '@lezer/common'
 import { forEachDiagnostic, type Diagnostic, forceLinting, setDiagnostics } from '@codemirror/lint'
 import { applyBold, applyItalic, insertLink, applyBlockquote, applyOrderedList, applyBulletList, applyTaskList } from '../commands/markdown'
 import { cut, copyAsPlain, copyAsHTML, paste, pasteAsPlain } from '../util/copy-paste-cut'
+// import { addNewFootnote } from '..'
```

Comment:
Remove this entire change as it only consists of a comment and superfluous newlines